auth-web/src/util/constants.ts

```diff
@@ -196,7 +196,7 @@ export enum NrState {
 export enum NrDisplayStates {
     APPROVED = 'Approved',
     HOLD = 'Pending Staff Review',
-    DRAFT = 'Draft',
+    DRAFT = 'Pending Staff Review',
```

Comment:
This feels funny, is there another field in combination with DRAFT and makes it so it's PENDING STAFF REVIEW?

Reply:
yeah looks funny, but it's from previous code, I thought it's Draft then lead to bug, I guess it's because backend do not have "pending staff review" status?

It's Draft from api, so if a nameRequest is not 'TMP' or 'RTMP', and status is 'Draft', it suppose display as 'Pending Staff Review'

Reply:
Then we should add a condition where this enum is called to check the other part of nameRequest and adjust it accordingly.

Reply:
sorry I just realized, I can't change it to DRAFT = 'Draft', here is an example from old api:
`
        {
            "affiliations": [
                {
                    "created": "2023-04-13T21:30:54+00:00",
                    "id": 18648
                }
            ],
            "businessIdentifier": "NR 4630418",
            "contacts": [],
            "corpType": {
                "code": "NR",
                "default": false,
                "desc": "Name Request"
            },
            "created": "2023-04-13T21:30:49+00:00",
            "lastModified": "2023-04-13T21:30:54+00:00",
            "modified": "2023-04-13T21:30:54+00:00",
            "name": "",
            "passCodeClaimed": true,
            "status": "DRAFT"
        }
`
it suppose display: 
![image](https://user-images.githubusercontent.com/16507223/234413426-f0d03688-361d-4806-9bed-08e73256958f.png)
Here is data from new api:
`
        {
            "legalType": "NR",
            "nameRequest": {
                "actions": [
                    {
                        "URL": null,
                        "entitiesFilingName": null,
                        "filingName": "Incorporation",
                        "learTemplate": null
                    },
                    {
                        "URL": null,
                        "entitiesFilingName": null,
                        "filingName": "Amalgamation",
                        "learTemplate": null
                    }
                ],
                "applicants": [
                    {
                        "emailAddress": "ashna.mehta@gov.bc.ca",
                        "phoneNumber": "9876543210"
                    }
                ],
                "consentFlag": null,
                "entityTypeCd": "CR",
                "expirationDate": null,
                "id": 2265751,
                "legalType": "BC",
                "names": [
                    {
                        "name": "PEAKS AND STEEPS CORPORATION",
                        "state": "NE"
                    }
                ],
                "natureBusinessInfo": "dsc",
                "nrNum": "NR 4630418",
                "requestTypeCd": "CR",
                "stateCd": "DRAFT",
                "target": "lear"
            }
        }
`

Reply:
sounds good